test/test_app.sh

```diff
@@ -33,18 +33,32 @@ end
 GEMFILE
 # For Rails 5 Gemfile
 sed -i'' -e "s/gem 'chromedriver-helper'/gem 'webdrivers'/" Gemfile
+# Rails 5.0 doesn't work with sqlite >= 1.4
+if [ "${RAILS_VERSION}" = "5.0" ]; then
+  sed -i'' -e "s/gem 'sqlite3'/gem 'sqlite3', '~> 1.3.6'/" Gemfile
+fi
 bundle update
 
 sed -i'' -e 's,rails/test_help,test/unit/rails/test_help,g' test/test_helper.rb
 # TODO: Implement.
 sed -i'' -e 's/parallelize/# parallelize/g' test/test_helper.rb
 
-if rails --version | grep -q "^Rails 6"; then
+if [[ "${RAILS_VERSION}" =~ ^6 ]]; then
   rails webpacker:install
 fi
 rails generate model item name:string
 sed -i'' -e 's/# //g' test/models/item_test.rb
 rails db:migrate
-rails test -v | tee test.log
-grep models: test.log
-grep assertions test.log
+
+if [ `echo "${RAILS_VERSION} >= 5.1" | bc` = 1 ]; then
```

Comment:
```suggestion
if [ $(echo "${RAILS_VERSION} >= 5.1" | bc) -eq 1 ]; then
```

Reply:
* `$(...)` is better than `` `...` `` in shell
* `=` is a string operator and `-eq` is a number operator in `[`

Reply:
Thank you for the review. Applied your suggestions and force-pushed.